### DISCUSSION TITLE
How to Disable ScrollBar Customization of DaisyUI

### DISCUSSION THREAD

**Discussion by Keshav-writes-code** [saadeghi/daisyui#3113] [Q&A] [answered]
So, i have a global.css file in my project where i defined some custom properties for the scrollbars 

`@tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
  .scrollbar::-webkit-scrollbar {
    width: 3px;
  }

  .scrollbar::-webkit-scrollbar-track {
    border-radius: 100%;
    background: #02050d;
  }

  .scrollbar::-webkit-scrollbar-thumb {
    background: #e0cbcb;
    border-radius: 100vh;
    border: 3px solid #f6f7ed;
  }

  .scrollbar::-webkit-scrollbar-thumb:hover {
    background: #c0a0b9;
  }
}`

so, when i installed DaisyUi, the scrollbar was all set to some other default Properties
i dont want that
i want mine properties to be applied

**Comment by flomero** [comment, marked as answer]
tailwind changes the `scrollbar-color` property of everything on your page. can you try adding this to remove the tailwind styles

```css
*,
*:hover {
    scrollbar-color: auto;
}
```

> If [scrollbar-color](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color) and [scrollbar-width](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) are supported and have any value other than auto set, they will override ::-webkit-scrollbar-* styling. See [Adding a fallback for scrollbar styles](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar#adding_a_fallback_for_scrollbar_styles) for more details. 

see: https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

**Reply by Keshav-writes-code** [reply]
Thanks

**Comment by vuhoanglam** [comment]
I'm facing this exact issue, and I can't believe something this simple has taken almost 2 hours and still can't be resolved. I want daisyUI to not interfere with my custom scrollbar styles. It has absolutely nothing to do with Tailwind CSS. When I remove the daisyUI plugin, everything works fine.

@tailwind base;
@tailwind components;
@tailwind utilities;

body {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}

::-webkit-scrollbar {
    width: 6px;
    height: 6px;
}

::-webkit-scrollbar-track {
    background: transparent;
}

::-webkit-scrollbar-thumb {
    background: rgba(200, 200, 200, 0.8);
    border-radius: 3px;
}

::-webkit-scrollbar-thumb:hover {
    background: rgba(180, 180, 180, 0.7);
}

**Reply by saadeghi** [reply]
`::-webkit-scrollbar` is not CSS standard and will be removed form webkit browsers in favor of the new, standard `scrollbar-color` so I wouldn't suggest using `scrollbar-color` instead of `::-webkit-scrollbar`

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

>Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

But if you want to, you can disable daisyUI `base` style from config.

https://daisyui.com/docs/config/#base

https://github.com/saadeghi/daisyui/tree/master/src/base

**Reply by abs** [reply]
Just a heads up that `base: false` isn't working in 5 beta - custom scrollbar styles get overwritten

**Comment by vuhoanglam** [comment]
![image](https://github.com/user-attachments/assets/8bc39179-b353-49aa-9ff7-43d8e5c5a248)
you are the best, disable base style do the trick.

I don't know why, but when I manually set the width for webkit-scrollbar, the top and bottom arrows disappear and make it look much better, which is exactly what I want. If I use scrollbar-color and scrollbar-width, it immediately makes the arrows reappear at both ends of the scrollbar and the gutter will also have padding

::-webkit-scrollbar {
width: 6px;
height: 6px;
}

Image is zoomed 400%
You can check this link: https://playcode.io/2094263
![image](https://github.com/user-attachments/assets/25896c0f-f00e-4bbd-829c-e67ca0c006b0)
